src/lib/utils/enum.utils.ts

```diff
@@ -0,0 +1,7 @@
+export const enumFromStringExists = <T>({
```

Comment:
Just thinking, will this add any value here?

```suggestion
export const enumFromStringExists = <T extends object>({
```

Reply:
Not sure but I like it 😉